### PR TITLE
Update GitHub actions for CI

### PR DESCRIPTION
**Pull request by afbora**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -301,7 +301,7 @@ jobs:
 
       - name: Check for unused Composer dependencies
         if: always() && steps.finishPrepare.outcome == 'success'
-        run: composer unused --no-progress
+        run: composer-unused --no-progress
```
Comment (afbora) [inline .github/workflows/ci.yml:304]: No longer works as `composer unused`. Please see the log:

https://github.com/getkirby/kirby/runs/5146628141?check_suite_focus=true

Comment (afbora) [reply .github/workflows/ci.yml:304]: Also I tried v8 but failed, see logs:
https://github.com/getkirby/kirby/runs/5146326654?check_suite_focus=true

Comment (distantnative) [reply .github/workflows/ci.yml:304]: But seems to work with your change, right?

Comment (afbora) [reply .github/workflows/ci.yml:304]: Yes works with current implementation but need to review from @lukasbestle to confirm

**Review by lukasbestle — APPROVED** [review]

**Pull request merged**